patch-resources.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+
```

Comment:
It would be great if this script or commit had a high-level description of what this file is, what needs to be changed, what problem is being solved, etc.